app/src/main/java/me/saket/dank/widgets/TextViewLimitSelection.java

```diff
@@ -0,0 +1,60 @@
+package me.saket.dank.widgets;
+
+import android.content.Context;
+import android.text.Selection;
+import android.text.Spannable;
+import android.util.AttributeSet;
+
+public class TextViewLimitSelection extends androidx.appcompat.widget.AppCompatTextView {
```

Comment:
Also something like `SelectionLimitingTextView` or similar could be a better name so TextView stays at end 🙂